extensions/sql-database-projects/src/dialogs/updateProjectFromDatabaseDialog.ts

```diff
@@ -5,7 +5,7 @@
 
 import * as vscode from 'vscode';
 import * as mssql from '../../../mssql';
-import * as azdata from 'azdata';
+import type * as azdata from 'azdata';
 import * as constants from '../common/constants';
 import * as newProjectTool from '../tools/newProjectTool';
 import type * as mssqlVscode from 'vscode-mssql';
```

Comment:
this vscode-mssql import shouldn't be needed since this is a dialog and there isn't support for update project from db in vscode yet.

Reply:
It's only being used in the constructor for the connection parameter - I tried to remove it but the controller that creates it is using some common functions (getConnectionProfileFromContext) which is typed to return a union of all these types. It's probably not worth trying to fix that at this point.